src/rebar.lock

```diff
@@ -0,0 +1 @@
+[].
```

Comment:
What is this?

Reply:
oh, someone executed `rebar compile` inside `src` xd